File: src/fizzBuzz.js
class FizzBuzz {
  play(number) {
    if (number % 3 === 0) {
      return "fizz";
    } else if (number % 5 === 0) {
      return "buzz";
    } else if (number % 15 === 0) {
      return "fizz-buzz";
    } else {
      return number;
    }
  }
}

module.exports = FizzBuzz;

File: src/fizz_buzz.rb
class FizzBuzz
  def play(number)
    if number % 3 == 0
      "fizz"
    elsif number % 5 == 0
      "buzz"
    elsif number % 15 == 0
      "fizz-buzz"
    elsif number = 1
        1
    end
  end
end
